src/Microsoft.DotNet.Wpf/src/System.Xaml/System/Xaml/Permissions/XamlLoadPermission.cs

```diff
@@ -17,6 +17,12 @@ public sealed class XamlLoadPermission : CodeAccessPermission, IUnrestrictedPerm
         private static IList<XamlAccessLevel> s_emptyAccessLevel;
         private bool _isUnrestricted;
 
+        private const string IPermissionTagName = "IPermission";
```

Comment:
Should all this CAS stuff be reduced to stubs, as we don't support CAS on .NET Core?

Reply:
Not sure. Probably not something for this PR, opened #241